R/calc_variables_tree_level.R

```diff
@@ -0,0 +1,110 @@
+#' Calculate additional variables on tree level
+#'
+#' This function calculates additional variables based on measurements, such as
+#' \itemize{
+#'  \item tree_number: the number of shoots in the tree (= 1 for an individual tree; >= 1 when coppice)
+#'  \item individual: true for individual tree or coppice, false if record is a secondary shoot
+#'  \item calc_height_m: calculated height based on `dbh_mm` and a species specific diameter-height model
+#'  \item basal_area_m2
+#'  \item vol_bole_m3: calculated based on `dbh_mm`, `calc_height_m` and species specific tariffs
+#'  \item vol_crown_m3: calculated based on `dbh_mm` and species specific tariffs
+#'  \item vol_tot_m3: sum of `vol_bole_m3` and `vol_crowwn_m3`
+#'  \item dbh_mm (based on average for coppice trees)
+#'  \item decaystage (based on average for coppice trees)
+#'  \item basal_area_alive_m2_ha
+#'  \item basal_area_dead_m2_ha
+#'  \item vol_alive_m3_ha
+#'  \item vol_dead_standing_m3_ha
+#'  \item vol_bole_alive_m3_ha
+#'  \item vol_bole_dead_m3_ha
+#' }
+#'
+#' @inheritParams calc_variables_stem_level
+#' @param data_dendro dataframe on tree measures with variables plot_id, plottype, tree_measure_id, date_dendro, dbh_mm, height_m, species, alive_dead, decaystage, period, OldID, year, subcircle, plotarea_ha,... (output of function load_data_dendrometry())
+#'
```

Comment:
```suggestion
#' @param data_dendro dataframe on tree measures with variables plot_id, plottype, tree_measure_id, date_dendro, dbh_mm, height_m, species, alive_dead, decaystage, period, OldID, year, subcircle, plotarea_ha,... (output of function load_data_dendrometry())
#' @param data_stems_calc ...
#'
```
Een argument vergeten (ook ontdekt in de check): vul jij de beschrijving aan?